src/LiveComponent/composer.json

```diff
@@ -37,6 +37,7 @@
         "doctrine/doctrine-bundle": "^2.4.3",
         "doctrine/orm": "^2.9.4",
         "doctrine/persistence": "^2.5.2|^3.0",
+        "matthiasnoback/symfony-dependency-injection-test": "^5.1",
```

Comment:
Is this absolutely necessary ?

Reply:
Of course it isn't. But I decided to add it as:
1) it makes testing compiler passes, extension and other things way easier
2) we're using it in Sylius, and it's a trustworthy dependency

If you want, I can remove it and test the compiler pass other way, but if we plan to create more tests covering extensions/compiler passes, it might be cool to leave it.

Reply:
I really like this bundle - as config/compiler passes get more complex it's super useful - let's keep it!